static/src/javascripts/projects/commercial/modules/prebid/types.js

```diff
@@ -36,11 +36,13 @@ export type PrebidXaxisParams = {
 export type PrebidAppNexusParams = {
     placementId: string,
     customData: string,
+    lotame?: {},
```

Comment:
Why will this be a maybe? We seem to always add it. I think the value could sometimes be undefined, if the `window. OzoneLotameData` is missing...

Reply:
We don't add it for appnexus direct for instance. I'll cover the undefined case :) Thanks!

Reply:
That's a good point actually, @jeteve are you happy with what should have lotame and what shouldn't? 

Happy to run though!

Reply:
Sure @ScottPainterGNM . As far as I know, this lotame account we have will be used by Ozone for audience management (the account will be shared across partners). So it should be only what goes to Ozone.